AST.fs

```diff
@@ -153,6 +168,22 @@ module Pretty =
     open Starling.Core.Model.Pretty
     open Starling.Core.Var.Pretty
 
+    /// Some ANSI color codes for nice outputting during debugging
+    /// wraps x as "<[line n: x]>" with color formats
+    let pprint_lines = false
+    let line_info_fmt x = sprintf "\u001b[92m<[line %d: \u001b[0m" x
```

Comment:
The other function names are camelCase, it'd be nice if these were consistent.  (I _think_ camelCase is the F# norm)